noah/src/main/resources/crawler/promed_mail.py

```diff
@@ -72,15 +54,40 @@ def getEachMail(archNo):
 
 if __name__ == '__main__':
     archNo = set()
-    end_date = datetime.date.today().strftime("%m/%d/%Y")
-    start_date = datetime.datetime(2007, 01, 01).strftime("%m/%d/%Y")
-    filename = "./promedsearchresult.html"
-    request(filename, getNext(start_date, end_date))
-    (listId) = getArchiveNosAndPages(filename)
+    headers = {}
+    headers['User-Agent'] = "Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/51.0.2704.103 Safari/537.36"
+    headers['X-Requested-With'] = 'XMLHttpRequest'
+    headers['Referer'] = "http://promedmail.org/"
+    headers['Accept'] = "application/json, text/javascript, */*"
+    headers['Accept-Encoding'] = "gzip, deflate, sdch"
+    headers['Accept-Language'] = "en-US,en;q=0.8"
+    headers['Connection'] = "keep-alive"
+    headers['Content-Type'] = "application/x-www-form-urlencoded"
+    headers['Host'] = 'promedmail.org'
+    
+    end_date = datetime.date.today()
+    start_date = end_date - datetime.timedelta(7)
+    end_date = end_date.strftime("%m/%d/%Y")
+    start_date = start_date.strftime("%m/%d/%Y")
+    filename = "promedsearchresult.html"
+    dirName = "PromedResult"
```

Comment:
That can be changed by changing the cron job time. the start date can be constant, i.e; the date we run the program. So we can make the time interval to be entered via cmd. such as 7 days or 2 days. That can be done. But the thing is, we are entering the website from the search tab which is not modified in real time. It gets modified after some time ( maybe days after the actual update). So, I feel that keeping the start date interval a constant value and just changing the end date by changing the cron time sounds a more user-friendly way. Even if we run it every day, it will show the results of 7 days from that date. And since we have already taken care of avoiding duplicates, it shouldn't matter. 


If we want the cmd parameters, I can add them as one. Its just a line of code.